Makefile

```diff
@@ -79,6 +79,7 @@ protoc: protodocs
 	protoc --gogofaster_out=. $(PROTOC_FLAGS) x/paychan/*.proto
 	protoc --gogofaster_out=. $(PROTOC_FLAGS) x/currency/*.proto
 	protoc --gogofaster_out=. $(PROTOC_FLAGS) x/gov/*.proto
+	protoc --gogofaster_out=. $(PROTOC_FLAGS) x/aswap/*.proto
```

Comment:
I think we could auto-detect proto files someday...
But that's for a different pr